packages/fxa-payments-server/src/store/reducers.ts

```diff
@@ -61,4 +62,4 @@ export const reducers = {
     },
     defaultState.api
   ),
-};
+} as ReducersMapObject<State, any>;
```

Comment:
Luckily, this looks like one of the only actual code changes required by the typescript upgrade